apps/test/unit/SoundsTest.js

```diff
@@ -1,11 +1,11 @@
 import {expect} from '../util/deprecatedChai';
 import Sounds from '@cdo/apps/Sounds';
 import sinon from 'sinon';
+import winMp3 from '!!file-loader!../audio/assets/win.mp3';
```

Comment:
I've fixed this by using Webpack's [file-loader](https://webpack.js.org/loaders/file-loader/) [inline](https://webpack.js.org/concepts/loaders/#inline) (which is nice because we don't really need this as part of our global config) to pull an existing fixture mp3 file from our repo, which is guaranteed to load unlike the broken URL below.